package.json

```diff
@@ -33,7 +33,11 @@
       "git add"
     ],
     "*.ts": [
-      "tslint -c tslint.json 'packages/**/*.ts'",
+      "tslint -c tslint.json 'packages/**/*.ts' --fix",
+      "git add"
+    ],
+    "*.tsx": [
+      "tslint -c tslint.json 'packages/**/*.ts' --fix",
```

Comment:
Shouldn't this be `"tslint -c tslint.json 'packages/**/*.tsx' --fix"`?

Reply:
There you go https://github.com/zeit/next.js/pull/6145

Reply:
Merged!